hack/ha.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
Same re `hack/util`.

Reply:
lol I have also realized now that "sh" has been omitted because that is the file extension.

Whilst cool, I think for completeness and reducing parsing time for humans, we should keep "sh" in the filename as well as the extension.

Reply:
> lol I have also realized now that "sh" has been omitted because that is the file extension.

:cry: I'd never have even thought that there might be places where the extension isn't rendered, it's so rare that I see that nowadays

I'll rename!